recipes/beyla/collector-config.yaml

```diff
@@ -116,8 +129,12 @@ spec:
         traces:
           receivers: [otlp]
           processors: [filter/httpserveronly]
-          exporters: [spanmetrics/httpserver]
+          exporters: [spanmetrics/httpserver, spanmetrics/httpservicegraph, googlecloud]
         metrics/spanmetrics:
           receivers: [spanmetrics/httpserver]
           processors: [groupbyattrs, resource, k8sattributes, resource/podinstance, filter/podonly, transform/prometheusconventions, resourcedetection]
           exporters: [logging, googlemanagedprometheus]
+        metrics/httpservicegraph:
+          receivers: [spanmetrics/httpservicegraph]
+          processors: [groupbyattrs, resource, k8sattributes, resource/podinstance, filter/podonly, transform/prometheusconventions, resourcedetection]
```

Comment:
If this is using the same processors and exporters, you can add the receiver to the same pipeline as spanmetrics above.